src/components/CopyUriButton/CopyUriButton.tsx

```diff
@@ -10,15 +10,18 @@ interface Props {
 function CopyUriButton({ uri }: Props) {
   const { t } = useTranslation('components', { keyPrefix: 'copy-uri-button' });
 
+  if (!uri) {
+    return <></>;
+  }
+
   return (
-    <>
-      {uri && (
-        <Group pt={'sm'}>
-          <Code>{t('label')}:</Code>
-          <CopyToClipboardButton value={uri} />
-        </Group>
-      )}
-    </>
+    <Stack pt={'sm'}>
+      <Group gap={'xs'} wrap={'nowrap'}>
+        <Text size={'xs'}>{t('label')}:</Text>
+        <Code>{uri}</Code>
+        <CopyToClipboardButton value={uri} />
+      </Group>
+    </Stack>
```

Comment:
```suggestion
      <Group pt={'sm'} gap={'xs'} wrap={'nowrap'}>
        <Text size={'xs'}>{t('label')}:</Text>
        <Code>{uri}</Code>
        <CopyToClipboardButton value={uri} />
      </Group>
```

Reply:
Indeed - it's a remnant from when I was experimenting with a different layout... Good catch :)